### PR TITLE
Automatic releasing

### PR DESCRIPTION
**Pull request by Morlack**
Adds automatic signing and publishing of plugin to Jetbrains marketplace when a tag is pushed.

It will not update the version/changelog. See "Development process" section in README.md

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Build and Publish
+
+on:
+    workflow_dispatch:
+    pull_request:
+    push:
+        branches:
+            - master
```
Comment (lfgcampos) [inline .github/workflows/build.yml:8]: we can also add `- main` to be future proof

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Publish
+
+on:
+    release:
+        types:
+            - created
```
Comment (lfgcampos) [inline .github/workflows/publish.yml:6]: We can also bind it to a tag push, up to you!
Also, we can add the `workflow_dispatch` if we need to trigger it manually

**Review by lfgcampos — APPROVED** [review]

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: actions/checkout@v2
+            -   name: Set up JDK 11
+                uses: actions/setup-java@v2
+                with:
+                    java-version: '11'
```
[the remark this answers is not in the document]
Comment (Morlack) [reply .github/workflows/build.yml:20]: We have to follow the bundled IntelliJ JDK for this, so 11 is the way to go

File: .github/workflows/build.yml (hunk 1 above)
Comment (Morlack) [reply .github/workflows/build.yml:8]: Great suggestion, I'll add it

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+name: Build and Publish
+
+on:
```
[the remark this answers is not in the document]
Comment (lfgcampos) [reply .github/workflows/build.yml:3]: already done =)

**Pull request merged**